### PR TITLE
Info panel and provincial summary

### PR DESCRIPTION
**Pull request by dgboss**
Adds a static side panel to ASA. The side panel includes a permanent ProvincialSummary component that is collapsible. It summarizes/correlates to the fire zone unit data you see on the map. Unfortunately there is no data in the dev deploy, so you'll have to spin this up locally if you have data, select a date of interest and forecast/actual.

The second panel will show the zone unit details when you click on a fire zone on the map.

Adds a schema migration and data migration to get the data we need to display.

# Test Links:
[Landing Page](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3608-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/fba/components/infoPanel/fireZoneUnitSummary.test.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import FireZoneUnitSummary from 'features/fba/components/infoPanel/FireZoneUnitSummary'
+import { FireShape } from 'api/fbaAPI'
+import { render } from '@testing-library/react'
+
+// fire - zone - unit - summary
+
+// selectedFireZoneUnit: FireShape | undefined
```
Comment (conbrad) [inline web/src/features/fba/components/infoPanel/fireZoneUnitSummary.test.tsx:8]: commented out code

**Review by conbrad — APPROVED** [review]
Amazing, love the tests! Just a few minor nits.

File: .vscode/settings.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
         "api/.venv": true
     },
     "editor.codeActionsOnSave": {
-        "source.fixAll.eslint": true
+        "source.fixAll.eslint": "explicit"
```
Comment (dgboss) [inline .vscode/settings.json:20]:
```suggestion
        "source.fixAll.eslint": true
```

File: api/alembic/versions/6ec6afd92af3_add_label_and_fire_centre_field_to_.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+"""Add label and fire centre field to advisory_shapes
+
+Revision ID: 6ec6afd92af3
+Revises: 2b3755392ad8
```
Comment (dgboss) [inline api/alembic/versions/6ec6afd92af3_add_label_and_fire_centre_field_to_.py:4]:
```suggestion
Revises: f9ab4e6a7eeb
```

File: api/alembic/versions/6ec6afd92af3_add_label_and_fire_centre_field_to_.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Create Date: 2024-04-30 10:22:37.880294
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+
+# revision identifiers, used by Alembic.
+revision = "6ec6afd92af3"
+down_revision = "2b3755392ad8"
```
Comment (dgboss) [inline api/alembic/versions/6ec6afd92af3_add_label_and_fire_centre_field_to_.py:14]:
```suggestion
down_revision = "f9ab4e6a7eeb"
```

**Review by brettedw — APPROVED** [review]
Really great work, looks awesome!

**Pull request merged**